admin/code/AdminRootController.php

```diff
@@ -9,28 +9,29 @@
 class AdminRootController extends Controller implements TemplateGlobalProvider {
 
 	/**
+ 	 * Fallback admin URL in case this cannot be infered from Director.rules
+	 *
+	 * @var string
+ 	 * @config
+	 */
+	private static $url_base = 'admin';
+
+	/**
 	 * Convenience function to return the admin route config.
 	 * Looks for the {@link Director::$rules} for the current admin Controller.
+	 *
+	 * @return string
 	 */
 	public static function get_admin_route() {
-		if (Controller::has_curr()) {
-			$routeParams = Controller::curr()->getRequest()->routeParams();
```

Comment:
We shouldn't be using the current controller because the current controller could be a non-admin one.